static/js/message_edit.js

```diff
@@ -305,6 +318,22 @@ function start_edit_maintaining_scroll(row, content) {
 }
 
 exports.start = function (row, edit_box_open_callback) {
+    if (Object.keys(currently_editing_messages).length === 0) {
```

Comment:
hmm, I think perhaps we should convert `currently_editing_messages` to use our `Dict` class and access that, for consistency with the rest of the codebase.